pkg/controller/namespace/namespace_controller.go

```diff
@@ -131,10 +131,10 @@ func (nm *NamespaceController) worker() {
 			return false
 		}
 
-		if estimate, ok := err.(*deletion.ResourcesRemainingError); ok {
-			t := estimate.Estimate/2 + 1
-			glog.V(4).Infof("Content remaining in namespace %s, waiting %d seconds", key, t)
-			nm.queue.AddAfter(key, time.Duration(t)*time.Second)
+		if _, ok := err.(*deletion.ResourcesRemainingError); ok {
+			//t := estimate.Estimate/2 + 1
+			//glog.V(4).Infof("Content remaining in namespace %s, waiting %d seconds", key, t)
+			nm.queue.AddAfter(key, 2*time.Second)
```

Comment:
what do we think the downside associated with this would be?  I always thought the estimate / 2 heruristic was arbirtrary.  What is the average estimate with current heuristic and should we just instead do 10% of estimate?